scripts/pipelines/build.core-feeds.sh

```diff
@@ -7,3 +7,12 @@ SCRIPT_ROOT=$(realpath $(dirname $BASH_SOURCE))
 echo "INFO: Building the core package feed."
 bitbake packagefeed-ni-core
 bitbake package-index
+
+# AB#2791248: Build the Linux Kernel to ensure CVEs are made available
+if [ "$1" == "next" ]; then
+   echo "INFO: Building the Linux Kernel Next"
+   bitbake linux-nilrt-next
+else
+   echo "INFO: Building the Linux Kernel"
+   bitbake linux-nilrt
+fi
```

Comment:
Why the conditionals? It shouldn't harm anything to ask bitbake rebuild them each time; right? It will simply skip everything in sstate.

Reply:
The `build.core-feeds.sh` script is currently executed as a part of `oe_feed_core` and `oe_feed_core_next`. The goal was to only build the relevant kernel for each feed.
Additionally, I wasn't sure we wanted the CVEs for linux-nilrt-next to be exported for `oe_feed_core` as it might have an impact on release

Reply:
I think `linux-nilrt-next` and `nilrt-next` are decoupled components. The -next _distribution_ doesn't always use the -next kernel. In fact, I think it never does. If the `linux-nilrt-next` kernel is currently in the extra/ packagefeed, then we don't need to have it in this pipeline script in any case, and can just build `linux-nilrt` for both the -next and current distribution components.